systems/x86_64-darwin/macbookpro/default.nix

```diff
@@ -1,20 +1,27 @@
-{
-  lib,
-  pkgs,
-  ...
+{ lib
+, pkgs
+, config
+, ...
 }: {
   # I'm not even sure these should all _be_ under the top-level config
   # The errors seem to indicate that as long as it's all either way...
   # But our module config has it under *config*.arichtman.default-home
   networking.hostName = "macbookpro";
-  #  I figured module name from directory is only used in flake.nix to apply the module
-  # arichtman.my_home.default-home = {
-  # my_home.default-home = {
-  default-home = {
```

Comment:
When in your system configuration, you can set Home Manager config for your user using `snowfallorg.user.<name>.home.config`.